.github/workflows/ci.yml

```
@@ -84,8 +84,8 @@ jobs:
         run: cargo test --verbose --no-default-features
       - name: Build with all features
         run: cargo test --verbose --all-features --no-run
-      - name: Tests with all features
-        run: cargo test --verbose --all-features
+      # - name: Tests with all features
+      #   run: cargo test --verbose --all-features
```

Replace this with a note about why we don't run the tests
```suggestion
      # Note: The `extension_inference` feature is experimental, we don't expect tests to succeed.
```